nginx-php/nginx.conf

```diff
@@ -59,6 +59,7 @@ http {
         "~*xing-contenttabreceiver"                 1;
         "~*chrome-lighthouse"                       1;
         "~*telegrambot"                             1;
+        "~*googlebot-inspectiontool"                1;
```

Comment:
i believe this should be `"~*google-inspectiontool"` just tried this configuration. ref: https://developers.google.com/search/docs/crawling-indexing/overview-google-crawlers#google-inspectiontool